Library/Formula/gdb.rb

```diff
@@ -24,13 +24,12 @@ class Gdb < Formula
     sha256 "1c48758042eed1ba9357fdb98b49b5b819555d664f9bb876e35c8bbaad58b0ec" => :mavericks
   end
 
-  option "with-brewed-python", "Use the Homebrew version of Python"
+  option "with-python", "Use the Homebrew version of Python; by default system Python is used"
```

Comment:
We're moving away from using `with-brewed-*` but this should have a `deprecated_option` added to flag this.